db/compaction/compaction_job.cc

```diff
@@ -1276,8 +1290,8 @@ Status CompactionJob::FinishCompactionOutputFile(
       auto kv = tombstone.Serialize();
       assert(lower_bound == nullptr ||
              ucmp->Compare(*lower_bound, kv.second) < 0);
-      sub_compact->AddToBuilder(kv.first.Encode(), kv.second,
-                                paranoid_file_checks_);
+      // Range tombstone is not supported by output validator yet.
+      sub_compact->builder->Add(kv.first.Encode(), kv.second);
```

Comment:
This fixes the problem with `paranoid_file_checks` + `DeleteRange()`, right? @mrambacher would it solve the problem to extract this into a separate PR that we can easily backport to 6.12 and 6.13?